.eslintrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  extends: '@mate-academy/eslint-config',
+  extends: "@mate-academy/eslint-config",
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0
+    "no-proto": 0,
```

Comment:
The `no-proto` rule is set to `0`, which disables it. If this is not the intended behavior, consider setting it to `1` (warn) or `2` (error) instead.